packages/gatsby/cache-dir/component-renderer.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from "prop-types"
 import loader, { publicLoader } from "./loader"
 import emitter from "./emitter"
 import { apiRunner } from "./api-runner-browser"
+import _ from "lodash"
```

Comment:
i'm not sure what the current preference is within the codebase but i usually prefer to pull in individual methods - e.g.

    import isEqual from 'lodash/isEqual'

see "installation" [here](https://github.com/lodash/lodash#installation)